### DISCUSSION TITLE
Builder - Be able to add only one time a type of block

### DISCUSSION THREAD

**Discussion by alepieser** [filamentphp/filament#7587] [Help]
Hello,

I would like to know if it's possible in v2 to force a builder to propose only one time to add a type block.

For example, I have 3 blocks: heading / paragraph / image. I would like that once we added the heading block, we propose only to add  paragraph or image.

**Comment by pboivin** [comment]
I'm not sure this is possible. Maybe you can implement this as a custom validation rule?
